database/mysql.js

```diff
@@ -123,11 +104,59 @@ const fs = require('fs');
       });
     };
 
+    var getAllFeatures = (page, callback) => {
+
+    };
+
+    var postFeatures = (features, callback) => {
+      if (isNaN(id)) {
+        callback('no Id to insert', null);
+      }
+      features.house_id = undefined;
+      features = keepOnlyFieldNames(features);

```

Comment:
features = Object.keys(features)

Why do you need features to be an object instead of an array?

if you really need it to be features = {... Object.keys(features)} Or features = (features) =>{
return features.each...}

Or use lodash _.forKey, you can import just that function. Hard coding field names/features, eek!